source/core/ut_suite_manager.pkb

```diff
@@ -290,15 +292,16 @@ create or replace package body ut_suite_manager is
       l_ind := g_schema_suites(a_owner_name).first;
       while l_ind is not null loop
         l_suite := g_schema_suites(a_owner_name) (l_ind);
-        l_suite.do_execute(a_reporter => a_reporter);
+        a_reporter := l_suite.do_execute(a_reporter => a_reporter);
```

Comment:
Decided to leave the only implementation of `do_execute` as procedure with `in no nocopy` reporter parameter.